### ISSUE TITLE
PointRend Model Outputs Incorrect Bounding Box: Bottom-Right Coordinates Are Incorrect

### ISSUE THREAD

**Issue by LAlongago** [facebookresearch/detectron2#5440]
I have trained a PointRend model using my custom dataset, and during inference, I noticed that the predicted bounding boxes have incorrect bottom-right coordinates. Specifically, the output format that should be `[x1, y1, x2, y2]` is being output as `[x1, y1, x2+w, y2+h]` . This means the model is adding the width w and height h to the correct x2 and y2 coordinates, which results in incorrect bounding box values.

The mask and class predictions are correct, but the bounding box coordinates are incorrect. Below is the inference code I am using:
```import torch
from detectron2.engine import DefaultPredictor
from detectron2.modeling import build_model
from detectron2.config import get_cfg
from detectron2.utils.visualizer import Visualizer
from detectron2.data import MetadataCatalog
from detectron2 import model_zoo
from detectron2.projects import point_rend
import cv2

MetadataCatalog.get("my_train_dataset").thing_classes = ["0-1", "1-2", "2-3", "3-4", "4-5", "5-6", "6-7", "7-8", "1-11", "1-12", "11-111", "11-112", "11-113", "111-1111", "111-1112", "111-1113", "112-1121", "112-1122", "113-1131", "113-1132", "2-21", "2-22", "22-221", "22-222", "22-223", "3-31", "4-41", "41-411", "41-412", "5-51", "51-511", "51-512", "6-61", "7-71", "8-81", "8-82"] 
MetadataCatalog.get("my_val_dataset").thing_classes = ["0-1", "1-2", "2-3", "3-4", "4-5", "5-6", "6-7", "7-8", "1-11", "1-12", "11-111", "11-112", "11-113", "111-1111", "111-1112", "111-1113", "112-1121", "112-1122", "113-1131", "113-1132", "2-21", "2-22", "22-221", "22-222", "22-223", "3-31", "4-41", "41-411", "41-412", "5-51", "51-511", "51-512", "6-61", "7-71", "8-81", "8-82"]

cfg = get_cfg()
point_rend.add_pointrend_config(cfg)
cfg.merge_from_file(model_zoo.get_config_file("COCO-InstanceSegmentation/pointrend_rcnn_R_50_FPN_1x_coco.yaml"))

cfg.DATASETS.TRAIN = ("my_train_dataset",)
cfg.DATASETS.TEST = ("my_val_dataset",)
cfg.DATALOADER.NUM_WORKERS = 4

cfg.MODEL.ROI_HEADS.NUM_CLASSES = 36
cfg.MODEL.POINT_HEAD.NUM_CLASSES = 36 
cfg.MODEL.ROI_HEADS.SCORE_THRESH_TEST = 0.5
cfg.SOLVER.IMS_PER_BATCH = 4 

cfg.MODEL.DEVICE = "cuda" 

cfg.MODEL.WEIGHTS = "model_final.pth" 

model = build_model(cfg)
checkpoint = torch.load(cfg.MODEL.WEIGHTS)
model.load_state_dict(checkpoint['model'], strict=False)

model.eval()
predictor = DefaultPredictor(cfg)

image = cv2.imread("WireInstanceDataset101/test/images/IMG20241018152403.jpg") 

outputs = predictor(image)
print(outputs)

v = Visualizer(image[:, :, ::-1], MetadataCatalog.get(cfg.DATASETS.TRAIN[0]), scale=0.8)
v = v.draw_instance_predictions(outputs["instances"].to("cpu"))

cv2.imshow("Inference Result", v.get_image()[:, :, ::-1])
cv2.imwrite("interface_output.jpg", v.get_image()[:, :, ::-1])
cv2.waitKey(0)
cv2.destroyAllWindows()
```
Could you please guide me on how to fix this issue with the bounding box coordinates? Is there any specific handling required in the code?

Thank you for your help!
